libbeat/outputs/redis/client.go

```diff
@@ -17,6 +18,9 @@ import (
 
 var (
 	versionRegex = regexp.MustCompile(`redis_version:(\d+).(\d+)`)
+
+	ackedEvents    = expvar.NewInt("libbeat.redis.published_and_acked_events")
```

Comment:
Yes, I kind of assumed you copied it from there. +1 on changing / reviewing all our naming for the metrics. We started this before we had our naming conventions, so most don't fit too well ...